src/Apps/Artwork/Utils/limitArrayOfStringsWithCount.tsx

```diff
@@ -0,0 +1,20 @@
+/**
+ * Helper function to limit array of strings for shorter displays
+ * @param initial array of strings to chech for the limit
+ * @param limit number of elements to keep
+ */
+export const limitArrayOfStringsWithCount = (
```

Comment:
Rather than encoding implementation details, could this be shortened to `limitWithCount`? 

Also, would you mind adding a small example in the docblock up above so that it its clear at a glance what this does? Something similar to what's seen here (though of course simpler): https://github.com/artsy/reaction/blob/master/src/DevTools/MockRelayRenderer.tsx#L38

(If it seems useful to others might want to place in the root `/Utils` folder)